sample/src/main/java/io/fotoapparat/sample/MainActivity.java

```diff
@@ -207,14 +210,22 @@ public void onError(CameraException e) {
     }
 
     private void takePicture() {
-        PhotoResult photoResult = fotoapparatSwitcher.getCurrentFotoapparat().takePicture();
+        final PhotoResult photoResult = fotoapparatSwitcher.getCurrentFotoapparat().takePicture();
 
         photoResult.saveToFile(new File(
                 getExternalFilesDir("photos"),
                 "photo.jpg"
         ));
 
         photoResult
+                .toPendingResult().whenAvailable(new PendingResult.Callback<Photo>() {
```

Comment:
I guess you added this to test the feature, but I would remove it from the example to keep it simple. Most of the users probably don't care how big picture is :)

Reply:
That's why I had added it. Removed from the sample now: 
https://github.com/Fotoapparat/Fotoapparat/pull/125/commits/143a3ad47cf985c30987dd71955e6edb677b0830